scripts/install-cni.sh

```diff
@@ -245,7 +246,7 @@ if [ "${ENABLE_CILIUM_PLUGIN}" == "true" ]; then
   echo "Cilium plug-in is in use. Holding CNI configurations until Cilium is ready."
 
   # inotify calls back to the beginning of this script.
-  inotify /host/home/kubernetes/bin cilium-cni "$0" cilium_ready
+  inotify /host/home/kubernetes/bin cilium-cni "$0" cni_ready cilium-cni
```

Comment:
I got lost here :)

Reply:
inotify uses argv+3 as callback, so the callback is `"$0" cni_ready cilium-cni`.

when calling back, `"$@"` is `cni_ready cilium-cni` so we're calling `cni_ready` with `cilium-cni` as `$1`.

Reply:
if you want a future person with that level of bash knowledge maintain this script you should write that down in the comment XD